data-serving/scripts/convert-data/parsers.py

```diff
@@ -239,6 +236,35 @@ def parse_location(value: Any) -> Dict[str, Any]:
     }
 
 
+def parse_location_list(value: Any) -> [Dict[str, Any]]:
+    '''
+    Parses one or more locations from a string.
+
+    Parameters:
+        value: Value representing a location or a delimiter-separated list of
+          locations.
+
+    Returns:
+      None: When the input value is empty or a string indicating n/a.
+      [Dict[str, Any]]: When the value is present and successfully parsed. 
+        A list of dictionaries containing the location data.
+    '''
+    separator = None
+    if pd.isna(value):
+        return None
+    if ';' in value:
+        separator = ';'
+    if ':' in value:
+        separator = ':'
+    if ' and ' in value:
+        separator = ' and '
+    if value.count(',') > 1:
```

Comment:
Making sure I understand this one: there are singular locations of the form "Miami, FL" -- but we're positive we've never inputted multiple comma-containing locations as a comma-delimited list?

Reply:
see below -- TL;DR: no, definitely not sure, and following PR will help track these down and log them